packages/web-components/src/components/rux-classification-marking/rux-classification-marking.tsx

```diff
@@ -84,6 +86,7 @@ export class RuxClassificationMarking {
                         'rux-classification--tag': type === 'tag',
                         'rux-classification--banner': type === 'banner',
                     }}
+                    part="container"
```

Comment:
This is only a container for the banner/tag, so maybe container isn't the best name for it?

Reply:
ya I agree. maybe header?